content/blog/2025-01-23-centos-connect-and-fosdem.md

```diff
@@ -0,0 +1,67 @@
+
+If you are in the area already or were already planning to be at FOSDEM, we'd love to say hi!
+
+## At CentOS Connect - Meetup and a talk!
+
+At Connect, other than enjoying connecting with other parts of our fellow CentOS Stream community, we're also contributing to the content twice:
+
+### ELevate meetup
+
+9 to 10:45 local time on Thursday, January 30, we'll be hosting a [meetup around Project ELevate](https://cfp.fedoraproject.org/centos-connect-2025/talk/TKT8TH/) - the tool that opens up in-place migrations between major versions across the enterprise Linux ecosystem. Upgrading between major versions of CentOS Linux was always a pain, and required tons of manual labor. 
+
+> Project ELevate is an open source upgrade tool that allows users to move between major versions of many of the most popular RHEL-equivalent operating systems without the tedium of moving content. Attendees will learn more about the project as well as how it can be used and improved.
+
+### AlmaLinux: the special derivative
+
+From 5 to 5:30 p.m. on Thursday Andrew will present, ["AlmaLinux: the special derivative"](https://cfp.fedoraproject.org/centos-connect-2025/talk/XTLCAX/). His talk will take everyone through the big differences that AlmaLinux brings to the enterprise linux world.
+
+> On the dates of CentOS Connect 2025, we get to celebrate exactly 4 years since the release of the very first beta version of AlmaLinux. While being RHEL (and later CentOS Stream) derivative AlmaLinux still does a lot of things differently both on the distribution and tooling sides. Our build system and mirror service are just the tip of the iceberg. In my talk I'd like to focus on more things we do differently, like:
+- How we build images
+- How we produce errata
+- How we do OpenSCAP profiles and OVAL data
+- How we support additional hardware and older CPUs
+- How we extend virtualization support
+
+> ... and many more.
+> This can be useful for users and developers to look at familiar features and processes from a new angle.
+
+
+## At FOSDEM - booth and Distros room and ANOTHER talk!
+
+At FOSDEM we'll be around to connect with folks at our booth on Saturday in building K, handing out some pens and stickers and saying hi to our friends. 
+
+Sunday, we'll be helping out in the [Distributions Dev Room](https://fosdem.org/2025/schedule/track/distributions/), as we go through the incredible content that's laid out for the day there (take a look if you haven't yet). 
+
+Andrew is again scheduled to give an AlmaLinux talk (from 12-12:30 p.m. local time): [“From Manul to Kitten: 4 years of AlmaLinux Development Evolution.”](https://fosdem.org/2025/schedule/track/distributions/) 
+
+In this one Andrew will share our experience in developing their distribution and the related services, while also outlining how we made certain decisions depending on emerging needs and external factors in order to continue to deliver and improve a high-quality distribution.
```

Comment:
I'd say the 1st phrase should begin this way: "In this one, Andrew will share our experience in developing AlmaLinux ..."

Reply:
this is what I get for copying from a press release.